bin/dynamo-tables.js

```diff
@@ -0,0 +1,87 @@
+const NAMESPACE = 'warehouse';
+
+const tables = {
+  [`${NAMESPACE}-objects`]: {
+    KeySchema: [
+      {
+        AttributeName: 'name',
+        KeyType: 'HASH'
+      },
+      {
+        AttributeName: 'env',
+        KeyType: 'RANGE'
+      }
+    ],
+    AttributeDefinitions: [
+      {
+        AttributeName: 'name',
+        AttributeType: 'S'
+      },
+      {
+        AttributeName: 'env',
+        AttributeType: 'S'
+      }
+    ],
+    ProvisionedThroughput: {
+      ReadCapacityUnits: 5,
+      WriteCapacityUnits: 5
+    }
```

Comment:
This feels like it should be configurable, as it'll vary based on the specific needs of the organization using Warehouse.

Reply:
At the moment, this is just for local stuff. It really doesn't matter.

Reply:
I mean these stuff is to create tables locally on local stuff for now.